security/cmd/node_agent_k8s/main.go

```diff
@@ -172,10 +176,17 @@ var (
 // Although currently not used, Citadel Agent can serve both workload and gateway secrets at the same time.
 func newSecretCache(serverOptions sds.Options) (workloadSecretCache, gatewaySecretCache *cache.SecretCache) {
 	if serverOptions.EnableWorkloadSDS {
+		var tlsRootCert []byte
+		if len(serverOptions.VaultTLSRootCert) > 0 {
+			tlsRootCert = []byte(serverOptions.VaultTLSRootCert)
+		} else {
```

Comment:
For readability, should SpireTLSBootstrapCert be placed in an else if statment? E.g.,
if len(serverOptions.VaultTLSRootCert) > 0 {
    ....
} else if len(serverOptions.VaultTLSRootCert) > 0 {
    ....
} else {
    ....
}